server/chat-plugins/room-faqs.js

```diff
@@ -47,8 +47,17 @@ const commands = {
 		topic = toId(topic);
 		if (!(topic && rest.length)) return this.parse('/help roomfaq');
 		let text = rest.join(',').trim();
+		// Strip out any potential links. The Perl 6 equivalent to this regex
+		// is included here since it's easier to read:
+		// [
+		// | '[[' [ <-[']']> | ']' <!before ']'> ]* ']]'
+		// | [ [ https? | ftp ] '://' ]?
```

Comment:
imo regular links shouldn't be stripped, since they appear verbatim anyway

Reply:
So they should be included when checking the length?

Reply:
Like, the length should be the literal actual length.

`[[this is a test <google.com>]]`

should be treated as

`this is a test`

and

`google.com`

should be treated as

`google.com`